lib/mix/lib/releases/assembler.ex

```diff
@@ -810,6 +810,9 @@ defmodule Mix.Releases.Assembler do
       {:mkdir, "releases/<%= release_version %>/commands"} |
       Enum.map(release.profile.commands, fn {name, path} ->
         {:copy, path, "releases/<%= release_version %>/commands/#{name}.sh"}
+      end) ++ #Hack. This allows for .bat stuff to be run on windows (instead of just forcing .sh)
```

Comment:
We should rethink how we implement this - things will explode if shell scripts are invoked as `.bat` (and vice versa). I think the best approach is to instead of forcing the extension, require paths to include the extension, and then in the platform-specific scripts, only invoke those extensions which we know we can invoke on that platform (i.e. only invoke `.bat` or `.ps` on Windows).

Reply:
@SuperDisk have you had a chance to look at this?

Reply:
@bitwalker I'll come check this out later tonight if I can.